packages/frontend/src/hooks/useProjects.ts

```diff
@@ -11,18 +10,18 @@ const getProjectsQuery = async () =>
     });
 
 export const useProjects = () => {
-    const setErrorResponse = useQueryError();
     return useQuery<OrganizationProject[], ApiError>({
         queryKey: ['projects'],
         queryFn: getProjectsQuery,
-        onError: (result) => setErrorResponse(result),
```

Comment:
Does this change how errors are handled?

Reply:
Yes. We don't want this error to show in the errors drawer. 
In some cases we want to fail silently and in other cases we want to show a error message in the page itself.